Tasks/KubernetesManifestV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -74,5 +77,6 @@
   "loc.messages.InvalidPromotetActionDeploymentStrategy": "Promote action works only with strategy: canary",
   "loc.messages.AllContainersNotInReadyState": "All the containers are not in a ready state.",
   "loc.messages.CouldNotDeterminePodStatus": "Could not determine the pod's status due to the error: %s",
-  "loc.messages.KubectlShouldBeUpgraded": "kubectl client version equal to v1.14 or higher is required to use kustomize features."
+  "loc.messages.KubectlShouldBeUpgraded": "kubectl client version equal to v1.14 or higher is required to use kustomize features.",
+  "loc.messages.TrafficSplitObjectDeleteFailed": "Unable to delete traffic split object: %s."
```

Comment:
For which cases deletion will fail? 
If we know the reason for failure, can we add that information in the message?